### DISCUSSION TITLE
Actually what is most popular React or Angular or Vue

### DISCUSSION THREAD

**Discussion by cesaracjota** [community/community#33313] [Other Feature Feedback, Questions, & Ideas]
Actually what is most popular React or Angular or Vue?

**Comment by melvincwng** [comment]
@CesarAcjotaMerma Hi Cesar, based on StackOverFlow, React is the most popular JavaScript library among the 3 options you've listed.

![Trends graph](https://user-images.githubusercontent.com/643434/187889624-bccb0672-076c-4162-bed6-88b62b7ce06c.png)

Also, based on StackOverFlow's survey in June:

> Popularity: React.js 42.62%, Angular 20.39%, Vue.js 18.82%, Angular.js 8.99%, Svelte 4.58%

Hope this clarifies and answers your question 😄 

Sources: 
1. https://insights.stackoverflow.com/trends?tags=reactjs%2Cvue.js%2Cangular%2Csvelte%2Cangularjs%2Cvuejs3
2. https://survey.stackoverflow.co/2022/

**Reply by aditamairham66** [reply]
but often when the creation of the frontend must be adjusted to the needs, for example if we want to make an application that must have the ability to process data or patterns that are quite complicated then we must also be able to choose the platform used

**Comment by xmxv** [comment]
React is definitely more popular.

**Comment by iLakshya** [comment]
This might be helpful:

https://medium.com/swlh/angular-vs-react-vs-vue-802a7c5f7e50
<img width="928" alt="Screenshot 2022-09-17 at 17 56 21" src="https://user-images.githubusercontent.com/91900978/190856729-54ccfc9a-351f-4ee1-8b41-285a08d50c96.png">
<img width="929" alt="Screenshot 2022-09-17 at 17 56 43" src="https://user-images.githubusercontent.com/91900978/190856732-26ea73bc-5292-4ed1-84d0-dc9fc6e43159.png">
<img width="1061" alt="Screenshot 2022-09-17 at 17 57 00" src="https://user-images.githubusercontent.com/91900978/190856733-559cc50b-0550-4372-9c11-9424d761172c.png">

**Comment by lebedevainna** [comment]
Hi. Here is a research on React vs Vue popularity I did for my article ![react vue popularity (1)](https://user-images.githubusercontent.com/87981162/199718441-17a74381-2d8b-49c0-a04b-8a94a52cf523.png)
"For now, React has a bigger community as it is backed by Facebook. [JetBrains survey](https://www.jetbrains.com/lp/devecosystem-2021/javascript/) says the React users’ share is 49%. But Vue’s community is also growing - users' share increased to 43% in 2021. 

Another indicator of popularity is the demand for the technology on Upwork, which you can see in the number of jobs where clients are looking for React vs Vue specialists. At the time of writing this article, there were 3,987 React jobs and 956 Vue jobs.

According to [W3Techs](https://w3techs.com/technologies/comparison/js-react,js-vuejs), 3% of all websites use React, and 0.7% - Vue. Their market share is 3.6% and 0.9% correspondingly.

Chrome Web Store shows that React has over [3,000,000](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi?hl=en) devtools users and Vue has over [1,000,000](https://chrome.google.com/webstore/detail/vuejs-devtools/nhdogjmejiglipccpnnnanhbledajbpd?hl=en) devtools users.

A [Stack Overflow survey](https://survey.stackoverflow.co/2022/#section-most-popular-technologies-web-frameworks-and-technologies) says that React is in second place among most common web technologies. It is used by 42.6% of developers. Vue is in sixth place. It is used by 18.8% of developers. Interestingly, the positions of these technologies remain the same no matter if developers are experienced or beginner, there is only a percentage change.

This survey also reveals that 5,559 developers who worked with React would like to try Vue; 6,492 who already worked with Vue want to continue to work with it."

If you need to compare other aspects (performance or SEO), here is a full version - https://greenice.net/vue-vs-react-which-is-better-according-to-the-experts-2022/
